src/libopensc/card-epass2003.c

```diff
@@ -1610,11 +1606,13 @@ epass2003_set_security_env(struct sc_card *card, const sc_security_env_t * env,
 		exdata->currAlg = SC_ALGORITHM_EC;
 		if(env->algorithm_flags & SC_ALGORITHM_ECDSA_HASH_SHA1)
 		{
+            sc_log(card->ctx, "setenva EC sha1 Algorithm alg_flags = %0x\n",env->algorithm_flags);
 			sbuf[2] = 0x91;
 			exdata->ecAlgFlags = SC_ALGORITHM_ECDSA_HASH_SHA1;
 		}
-		else if (env->algorithm_flags & SC_ALGORITHM_ECDSA_HASH_SHA256)
+	    else if (env->algorithm_flags & (SC_ALGORITHM_ECDSA_HASH_SHA256 | SC_ALGORITHM_ECDSA_HASH_NONE | SC_ALGORITHM_ECDSA_HASH_SHA1))
```

Comment:
Please, drop the needless whitespace change on this line. We use tabs for indentation.